css/src/components/toggle.scss

```diff
@@ -1,6 +1,6 @@
 $toggle-transition: $input-transition-duration cubic-bezier(0.01, 1.23, 0.58, 0.96) !default;
-$toggle-circle-size: 0.75em !default;
-$toggle-circle-top: 0.1875em !default;
+$toggle-circle-size: 0.85em !default;
```

Comment:
similar should be done for `radio` component:

https://design.learn.microsoft.com/components/radio.html

![image](https://github.com/user-attachments/assets/8102a4f2-93c3-4906-8ae6-a1b6eff60638)

vs Atlas:

![image](https://github.com/user-attachments/assets/5b6eda37-e7af-436a-8e8a-51344ce9b01a)

Reply:
updated